src/cdk/v2/destinations/smartly/utils.js

```diff
@@ -17,25 +17,25 @@ const {
  * @param {*} payload
  * @returns
  */
-const getPayloads = (event, eventsMapping, payload) => {
+const getPayloads = (event, Config, payload) => {
   if (!isDefinedAndNotNull(event) || typeof event !== 'string') {
     throw new InstrumentationError('Event is not defined or is not String');
   }
-  const eventsMap = getHashFromArrayWithDuplicate(eventsMapping);
+  const eventsMap = getHashFromArrayWithDuplicate(Config.eventsMapping);
   // eventsMap = hashmap {"prop1":["val1","val2"],"prop2":["val2"]}
   const eventList = Array.isArray(eventsMap[event.toLowerCase()])
     ? eventsMap[event.toLowerCase()]
     : Array.from(eventsMap[event.toLowerCase()] || [event]);
 
-  const payloadLists = eventList.map((ev) => ({ ...payload, event: ev }));
+  const payloadLists = eventList.map((ev) => ({ ...payload, event_name: ev }));
   return payloadLists;
 };
 
 const buildResponseList = (payloadList) =>
   payloadList.map((payload) => {
     const response = defaultRequestConfig();
     response.body.JSON = payload;
-    response.endpoint = config.singleEventEndpoint;
+    response.endpoint = config.batchEndpoint;
```

Comment:
@shrouti1507 We are using singleEventEndpoint for processor and we are specifically using this function for procWorkflow. So, here batchEndpoint should not be added. Should I revert this ?